docs/application/native/guides/multimedia/raw-audio.md

```diff
@@ -79,8 +81,10 @@ Before recording audio, you must define the following PCM data settings:
 - Audio sample type:
   - Unsigned 8-bit PCM
   - Signed 16-bit little endian PCM
+  - Signed 24-bit little endian PCM
+  - Signed 24-bit packed in 32-bit little endian PCM
 - Audio sample rate:
-  - 8000 ~ 48000 Hz
+  - 8000 ~ 192000 Hz
```

Comment:
You don't need to write this paragraph(from line 76 and line 87) again, because the content is over there.

How about changing like below?

---
Before recording audio, you must define the PCM data settings. For more information, see  [Audio Output](#play_pcm).

Reply:
updated done